pkg/azureutils/azure_disk_utils.go

```diff
@@ -219,7 +218,7 @@ func GetCloudProviderFromClient(kubeClient *clientset.Clientset, secretName stri
 	if kubeClient != nil {
 		klog.V(2).Infof("reading cloud config from secret %s/%s", az.SecretNamespace, az.SecretName)
 		az.KubeClient = kubeClient
-		config, err = az.GetConfigFromSecret()
+		config, err := az.GetConfigFromSecret()
```

Comment:
We need to fix this. This is from our my old changes that I fixed before merging with master.

Reply:
master has the correct changes for this function.